Manifests/ManagedPreferencesApplications/net.glencode.Particulars.plist

```diff
@@ -647,6 +647,6 @@
 	<key>pfm_unique</key>
 	<true/>
 	<key>pfm_version</key>
-	<integer>2</integer>
+	<integer>3</integer>
```

Comment:
Since no content was added and just a couple tweaks were made, we can leave the pfm_version the same as before and just update the last modified date as you have done

Reply:
The version has been updated as requested